nilearn/regions/tests/test_parcellations.py

```
@@ -171,32 +178,35 @@ def test_check_parameters_transform(test_image_2):
     assert imgs == fmri_imgs
     assert confounds_list == confounds
     # Test the error when length of images and confounds are not same
-    msg = ("Number of confounds given does not match with the "
-           "given number of images")
+    msg = (
+        "Number of confounds given does not match with the "
+        "given number of images"
+    )
 
     not_match_confounds_list = [confounds] * 2
     with pytest.raises(ValueError, match=msg):
         _check_parameters_transform(fmri_imgs, not_match_confounds_list)
 
 
-@pytest.mark.parametrize('method', METHODS)
-@pytest.mark.parametrize('n_parcel', [5])
-def test_parcellations_transform_with_multi_confounds_multi_images(method, n_parcel, test_image_2):  # noqa: E501
+@pytest.mark.parametrize("method", METHODS)
+@pytest.mark.parametrize("n_parcel", [5])
+def test_parcellations_transform_with_multi_confounds_multi_images(
+    method, n_parcel, test_image_2
+):  # noqa: E501
```

Remove ignore error statement

---

done